modules/ingest-common/build.gradle

```diff
@@ -18,6 +18,11 @@ dependencies {
   compileOnly project(':modules:lang-painless')
   api project(':libs:elasticsearch-grok')
   api project(':libs:elasticsearch-dissect')
+  api "org.apache.httpcomponents:httpclient:${versions.httpclient}"
+  api "org.apache.httpcomponents:httpcore:${versions.httpcore}"
+  api "commons-logging:commons-logging:${versions.commonslogging}"
+  api "commons-codec:commons-codec:${versions.commonscodec}"
+  api "org.apache.logging.log4j:log4j-1.2-api:${versions.log4j}"
```

Comment:
For both this and the above question, third-party audit complained until I explicitly added these and removed the `osgi` entries -- despite tests and builds passing.

Reply:
You should change the two `apache.httpcomponent` dependencies from `api` to `implementation` dependencies because they aren't needed by consumers of the ingest module. Those three other dependencies highlighted above can then be omitted and you will need only 'org.apache.commons.codec.binary.Base64', 'org.apache.commons.logging.Log', and 'org.apache.commons.logging.LogFactory' in the `ignoreMissingClasses` section.

Reply:
awesome, thanks for the info, I'll update soon.

Reply:
Thanks, that also means that you can remove the license and hash files for those dependencies from the PR.